src/helper_gitlab_modify.sh

```diff
@@ -448,40 +467,40 @@ checkout_branch_in_github_repo() {
   local github_repo_name="$1"
   local github_branch_name="$2"
   local company="$3"
-  
+
   if [ "$(github_repo_exists_locally "$github_repo_name")" == "FOUND" ]; then
 
     # Verify the branch exists
     branch_check_result="$(github_branch_exists $github_repo_name $github_branch_name)"
     last_line_branch_check_result=$(get_last_line_of_set_of_lines "\${branch_check_result}")
     if [ "$last_line_branch_check_result" == "FOUND" ]; then
-    
```

Comment:
I like how you delete unused whitespaces, is there a method you use to detect them? Or do you just delete them if your cursor happens to show them>?

Reply:
To be honest, the white spaces I found by coincidence and then delete them. However, with tabs, I'll search them with ctrl+f and replace them with space as is required by the google style sheet.

Reply:
First I looked at regexr.com That websites allows you to build different regular expressions using a cheatsheet, and then visualises the results of that expression and explains how the regular expression works.

regexr.com/6cihk shows you can select a group with brackets `()`, I think that means a specific order of characters (because newline+space does not match, but space+newline does match).  Then the `\s` represents a space character, and that could be followed by a newline character `\n` to find what you want to replace. I thought that could be used to replace ` \n` which is `\s\n` with: `\n`. However, doing that in notepad++ also removes arbitrary new lines. 

So I looked at: https://superuser.com/a/699386 and in the comments found that replacing `[ \t]+$` with nothing, works. However, that website says it should not match on anything regexr.com/6cihk , I do not exactly know why.

A simpler way would be to  `have bound the Ctrl+S key to the Macro -> Trim Trailing and save option, using the Settings -> Shortcut Mapper... option.` as Casey Kuball noted in that same answers' comments.